detection_rules/packaging.py

```diff
@@ -157,6 +166,7 @@ def save(self, verbose=True):
 
         if self.release:
             self.save_release_files(extras_dir, self.changed_rules, self.new_rules)
+            self._package_notice_file(rules_dir)
```

Comment:
Can you instead call this function from within `save_release_files`? Then it'll be impossible to save the files without also saving the notice.

Reply:
Oh, sorry. Realized this was within `save`.

Maybe just unindent this from the `self.release` check

Reply:
```suggestion

        self._package_notice_file(rules_dir)
```

Reply:
if a package is built with `release=False` then none of the release files or zips are built, just a directory of rules. If you are saying that you want it to always accompany _those rules_, then I can pull it up out of `if self.release:`

Reply:
yeah, exactly. If we save any of the rules, we should keep this with them, too